test/e2e/kibishii.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/pkg/errors"
+	"golang.org/x/net/context"
+)
+
+const (
+	kibishiiNamespace = "kibishii-workload"
+	jumpPadPod        = "jump-pad"
+)
+
+// runKibishiiTests runs kibishii tests on the provider.
+func runKibishiiTests(client testClient, providerName, veleroCLI, veleroNamespace, backupName, restoreName, backupLocation string,
```

Comment:
Why we move this function into the new file `kibishii.go`? I think it is also the util function called by other test cases?

Reply:
Yes, it's reorganized, thanks.